src/workspace.js

```diff
@@ -1532,6 +1536,12 @@ module.exports = class Workspace extends Model {
     ]
   }
 
+  getVisiblePaneContainers () {
```

Comment:
I'm intentionally excluding this method from the public API for now. We could consider making this method part of the public API in the future, or we could consider rolling this method's functionality into the existing `::getPaneContainers()` method. xref: https://github.com/atom/atom/pull/14640#discussion_r119168048